app/controllers/user.js

```diff
@@ -90,6 +91,22 @@ module.exports = {
       HANDLER.handleError(res, error)
     }
   },
+  // Load User
+  loadUser: async (req, res, next) => {
+    try {
+      const id = req.params.id || req.user._id
+      const user = await User.findById({ _id: id })
```

Comment:
As you are using findById no need to pass _id, just use findById(id);

Reply:
Oh, I missed that! I'll rectify it! 
Actually, I copied this piece of code from `userProfile` function which was written already. I'll rectify both!